server/src/main/java/org/opensearch/monitor/fs/FsInfo.java

```diff
@@ -235,6 +235,14 @@ public static class DeviceStats implements Writeable, ToXContentFragment {
         final long previousWritesCompleted;
         final long currentSectorsWritten;
         final long previousSectorsWritten;
+        long currentReadTime;
```

Comment:
so we have two options - either to make this final and initialize the fields to 0 explicitly for older versions. Or keep it non final. I guess i'll better make it final so that it looks consistent.